### PR TITLE
Grammar fix on Hacktoberfest page

### PR DESCRIPTION
**Pull request by krkahn**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5799"></a>

### CODE REVIEW

File: pages/docs/contributing/hacktoberfest.mdx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import DocsLayout from "@components/layouts/DocsLayout.js";
 
 ## Hacktoberfest
 
-Participate this annual event which encourages people to contribute to Open Source throughout the month of October by contributing to this project.
+Participate in this annual event which encourages people to contribute to Open Source throughout the month of October by contributing to this project.
```
Comment (CBID2) [inline pages/docs/contributing/hacktoberfest.mdx:5]:
```suggestion
Participate in this annual event by contributing to this project! 😄  
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @krkahn! 😄  I created a way to make the sentence clearer. I hope you find it helpful! 😄

File: pages/docs/contributing/hacktoberfest.mdx (hunk 1 above)
Comment (eddiejaoude) [reply pages/docs/contributing/hacktoberfest.mdx:5]: Thanks Chrissy but I prefer that it has "open source" in the sentence and also helps for seo

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍

**Pull request merged**